ccm_web/client/src/components/Home.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+import Grid from '@material-ui/core/Grid'
+
+import FeatureCard, { FeatureCardProps } from './FeatureCard'
+
+import AccountCircleOutlinedIcon from '@material-ui/icons/AccountCircleOutlined'
+import LibraryBooksOutlinedIcon from '@material-ui/icons/LibraryBooksOutlined'
+import MergeTypeIcon from '@material-ui/icons/MergeType'
+import PersonAddIcon from '@material-ui/icons/PersonAdd'
+import PersonAddOutlinedIcon from '@material-ui/icons/PersonAddOutlined'
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+    minWidth: 275
+  }
+}))
+
+function Home (): JSX.Element {
+  const classes = useStyles()
+
+  const mergeSectionProps: FeatureCardProps = {
+    title: 'Merge Sections',
+    description: 'Combine sections taught by yourself or others into one Canvas site for easier management',
+    icon: <MergeTypeIcon fontSize='large'/>,
+    size: 'tall'
+  }
+
+  const gradebookToolsProps: FeatureCardProps = {
+    title: 'Gradebook Tools',
+    description: 'Trim the gradebook from Canvas, or trim the gradebook from a third party to correct format',
+    icon: <LibraryBooksOutlinedIcon fontSize='large'/>,
+    size: 'short'
+  }
+
+  const addSectionsGroupsProps: FeatureCardProps = {
+    title: 'Add Sections / Groups',
+    description: 'Add sections, groups through csv viles into your own course',
+    icon: <AccountCircleOutlinedIcon fontSize='large'/>,
+    size: 'tall'
+  }
+
+  const addUMUsersProps: FeatureCardProps = {
+    title: 'Add UM Users',
+    description: 'Add UM users to your available sections',
+    icon: <PersonAddIcon fontSize='small'/>,
+    size: 'short'
+  }
+
+  const addNonUMUsersProps: FeatureCardProps = {
+    title: 'Add Non-UM Users',
+    description: 'Enroll non-UM users to your available sections',
+    icon: <PersonAddOutlinedIcon fontSize='small'/>,
+    size: 'short'
+  }
+
+  return (
+        <div className={classes.root}>
+            <Grid container spacing={3}>
+                <Grid item xs={12} sm={4}>
+                    <FeatureCard {...mergeSectionProps}/>
+                </Grid>
+                <Grid item xs={12} sm={4}>
+                    <Grid container spacing={3}>
+                        <Grid item xs={12}>
+                            <FeatureCard {...gradebookToolsProps}/>
+                        </Grid>
+                        <Grid item xs={12} sm={6}>
+                            <FeatureCard {...addUMUsersProps}/>
+                        </Grid>
+                        <Grid item xs={12} sm={6}>
+                        <FeatureCard {...addNonUMUsersProps}/>
+                        </Grid>
+                    </Grid>
+                </Grid>
+                <Grid item xs={12} sm={2}>
+                    <FeatureCard {...addSectionsGroupsProps}/>
+                </Grid>
+            </Grid>
+        </div>
```

Comment:
Looks like this is using a different indentation (4 spaces?) and some things aren't indented enough. I wonder if there's more linting configuration needed for JSX...

Reply:
What is  the  issue here?  I've noticed that the linter complains sometime about mismatched indentation, but I've always fixed it when it complains.

Do you know if it's expecting tabs or spaces?

Reply:
Indentation seems to be 4 spaces instead of 2 (which is enforced in the TS, I believe). I'd prefer to see that in the JSX as well? I also see at least one `FeatureCard` that needs to be indented again.

Reply:
The `standard` rule (first here: https://standardjs.com/rules.html#rules) is 2 spaces, if that helps.

Reply:
@ssciolla, In my IDE I configured to use the TypeScript Standard Style. I don't see MY IDE is  complaining with extra indentation. Am I missing anything?

![Screen Shot 2021-03-17 at 10 15 45 AM](https://user-images.githubusercontent.com/8579775/111481878-c2f19600-8709-11eb-8613-fdd2e3c5d26b.png)

Reply:
You are not missing anything. I am just wondering why the linting configuration is not complaining about the extra indentation. This isn't anyone's fault (except maybe mine), it's probably something that needs to be changed in `.eslintrc.js`.

Reply:
I think my editor is trying to use tabs by default.. my manual fix when the linter was complaining has been to remove the tab and put 2 spaces.  I guess  maybe its ending up with leading tabs and that's what you're seeing?
It's  not complaining on my end.  I'll see if i can figure out how to change vscode to use spaces.

Reply:
Hmm it was using spaces instead of tabs, just 4 instead of 2... but i was fixing it wherever i saw a complaint so not sure what the issue is.

Reply:
Yeah, in VSCode, you can change that by clicking where says "Spaces" on the bottom of the screen.